Makefile

```diff
@@ -13,6 +13,8 @@ prepare:
 	$(MAKE) check-links
 	cd os-autoinst && cpanm -nq --installdeps .
 	cpanm -nq --installdeps .
+	# ensure high enough limit for rename detection
+	git config diff.renameLimit || git config diff.renameLimit 4000
```

Comment:
This is an unrelated change

Reply:
Well unrelated to the one git commit, yes. But I put it into a separate git commit: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/12972/commits/c0e3e8d1426f09b719c5259d1934471a8e923832